src/classes/services/content_extractor.py

```
@@ -0,0 +1,232 @@
+            Domain string
+        """
+        try:
+            parsed = urlparse(url)
+            domain = parsed.netloc
+            
+            # Remove port number if present
+            if ':' in domain:
+                domain = domain.split(':')[0]
+                
+            return domain
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)